config/user-defined-memory-toy-problem.yaml

```diff
@@ -1,21 +1,20 @@
 # Specify input
 from_data:
-  data_stem: "../data/user-defined-memory-toy-problem/toy_mem"
+  data_stem: ../data/user-defined-memory-toy-problem/toy_mem
   phase_ids:
-    - 0
+  - 0
 
 # Specify work model
 work_model:
   name: AffineCombination
   parameters:
-    alpha: 1.
-    beta: 0.
-    gamma: 0.
+    alpha: 1.0
+    beta: 0.0
+    gamma: 0.0
     upper_bounds:
-      max_memory_usage: 8.0e+9
+      max_memory_usage: 8000000000.0
```

Comment:
Why this format?

Reply:
The file has been rewritten by the yaml dumper like all the configuration files. 
After some research it is due to Python standard formatting. In fact, Python will display values as scientific notation when they are bigger than 1.0e+15 or smaller than 1.e-4 this is internal to python formatting. And pyyaml output format is just the standard format. If you have a value of 8.0e+16 for example it will be formatted using the scientific notation.